app/src/main/java/com/example/currencyconverter/ui/CurrencyConverterApplication.kt

```diff
@@ -1,7 +1,7 @@
 package com.example.currencyconverter.ui
```

Comment:
NIT: It is not exactly UI piece - you can just put the App class into the base package (com.example.currencyconverter)